.goreleaser.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+# Make sure to check the documentation at https://goreleaser.com
+builds:
+  - env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -X github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/cmd.Version={{ .Version }}
+    goos:
```

Comment:
i believe this option `goos` now is not required since it's the default one.

Reply:
Done